### PR TITLE
Step01 Create a new activity and transferring data via Intent

### PR DESCRIPTION
**Pull request by b01001011**
Hey, Hans, I have done the first assignment.

The Step01 assignment was to
- List up items on a `ListView`.
- When click on a item, start a new `Activity`.
- The information about the clicked item, should be delivered via `Intent`.

Feel free to giving me a feedback on any aspect. 
Thanks!

### CODE REVIEW

File: Step01/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
```
Comment (HQHAN) [inline Step01/.gitignore]: @kwoolytech please take the standard .gitignore file for android here as there are some files that are not required to be shared.

https://github.com/github/gitignore/blob/master/Android.gitignore

File: Step01/app/src/main/java/com/kwoolytech/step01/DescriptionActivity.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            case "GE Appliances":
+                resourceId = R.drawable.geappliances;
+                break;
+        }
+
+        BitmapDrawable bitmapDrawable = (BitmapDrawable)ContextCompat.getDrawable(this, resourceId);
+        Bitmap bitmap = bitmapDrawable.getBitmap();
+
+        ImageView imageView = (ImageView)findViewById(R.id.imageView);
+        imageView.setImageBitmap(bitmap);
```
Comment (HQHAN) [inline Step01/app/src/main/java/com/kwoolytech/step01/DescriptionActivity.java]: @kwoolytech ImageView supports "setImageDrawable", you can set it to ImageView directly for simplicity.
http://developer.android.com/intl/ko/reference/android/widget/ImageView.html#setImageDrawable(android.graphics.drawable.Drawable)

**Comment by HQHAN** [comment]
Main purpose for this assignment is to gain understanding on "Intent" , "ListView" ( most prevalent UI component) in android. Let's talk about these two concepts sometime next week at the office.

You actually have met the requirement for the assignment very well.
Great job @kwoolytech !

Good to merge~!

File: Step01/app/src/main/java/com/kwoolytech/step01/DescriptionActivity.java (hunk 2 above)
Comment (b01001011) [inline Step01/app/src/main/java/com/kwoolytech/step01/DescriptionActivity.java]: Good simplification! Good to know. Thanks!

Comment (b01001011) [inline Step01/app/src/main/java/com/kwoolytech/step01/DescriptionActivity.java]: Cool. Updated. 👍

**Comment by b01001011** [comment]
@HQHAN Hey Hans, thank you for spare your time on review. I think I have address all of your comments. If it looks alright, merge please.

**Comment by HQHAN** [comment]
merging after code review

**Comment by HQHAN** [comment]
@kwoolytech just realized that i don't have permission to merge the code.
Give me a write access to this repo.

**Comment by b01001011** [comment]
Hey, @HQHAN, I've added you as a collaborator. Should able to merge. :)

**Comment by HQHAN** [comment]
Thanks keep up the good work!
1. 5. 10. 오후 9:27에 "Harry" notifications@github.com님이 작성:

**Pull request merged**